netstandard/ref/System.Reflection.Emit.cs

```diff
@@ -392,13 +475,31 @@ public sealed partial class MethodBuilder : System.Reflection.MethodInfo
         public void SetImplementationFlags(System.Reflection.MethodImplAttributes attributes) { }
 //      [System.ObsoleteAttribute("An alternate API is available: Emit the MarshalAs custom attribute instead. http://go.microsoft.com/fwlink/?linkid=14202")]
 //      public void SetMarshal(System.Reflection.Emit.UnmanagedMarshal unmanagedMarshal) { }
+        public void SetMethodBody(byte[] il, int maxStack, byte[] localSignature, System.Collections.Generic.IEnumerable<System.Reflection.Emit.ExceptionHandler> exceptionHandlers, System.Collections.Generic.IEnumerable<int> tokenFixups) { }
         public void SetParameters(params System.Type[] parameterTypes) { }
         public void SetReturnType(System.Type returnType) { }
         public void SetSignature(System.Type returnType, System.Type[] returnTypeRequiredCustomModifiers, System.Type[] returnTypeOptionalCustomModifiers, System.Type[] parameterTypes, System.Type[][] parameterTypeRequiredCustomModifiers, System.Type[][] parameterTypeOptionalCustomModifiers) { }
 //      Excluded because we don't support generating with debug information.
 //      public void SetSymCustomAttribute(string name, byte[] data) { }
         public override string ToString() { throw null; }
     }
+    public sealed partial class MethodRental
```

Comment:
MethodRental is very broken API (it was made public by accident .NET Framework 1.0). It should be definitely not exposed.

Reply:
BTW: Mono does not implement it either: https://github.com/mono/mono/blob/82e573122a55482bf6592f36f819597238628385/mcs/class/corlib/System.Reflection.Emit/MethodRental.cs#L59